yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryConfig.java

```diff
@@ -374,4 +392,22 @@ public static QueryConfig parse(@Nonnull Object object, @Nonnull YamlExecutionCo
             throw executionContext.wrapContext(e, () -> "‼️ Error parsing the query config at line " + lineNumber, "config", lineNumber);
         }
     }
+
+    public static class SkipConfig extends QueryConfig {
+        private final String message;
+
+        public SkipConfig(final String configMap, final Object value, final int lineNumber, final YamlExecutionContext executionContext, final String message) {
+            super(configMap, value, lineNumber, executionContext);
+            this.message = message;
+        }
+
+        @Override
+        void checkResultInternal(@Nonnull final Object actual, @Nonnull final String queryDescription) throws SQLException {
+            logger.info("Line: " + getLineNumber() + " " + message);
```

Comment:
Same.

This should probably be through a `RuntimeException` - which will ensure that this config is never run for a query. It's mostly precautionary.

Reply:
Done.